frontend/src/components/gotEnzymes/DetailsPage.vue

```diff
@@ -74,10 +72,6 @@ export default {
   },
   methods: {
     async setup() {
-      if (!SUPPLEMENTARY_INFO_TYPES.includes(this.componentType)) {
-        this.$store.dispatch(`gotEnzymes/resetInfoAndCrossReferences`);
```

Comment:
I suspect that this line is still needed, but maybe without the`return`?

If you click gene `zwf` from `gotenzymes/`, go back andt then visit domain `eukaryotes`, the cross references from the gene will still be shown. The provided links give blank pages / error messages.

Reply:
(this problem might already be covered by the comments from @e0 above)

Reply:
I'm quite sure @e0's commit [42ce1b7](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1062/commits/42ce1b74bae43c2253e4497d9c7fea20ea136089) resolves this.